lib/abr/simple_abr_manager.js

```diff
@@ -198,6 +204,24 @@ shaka.abr.SimpleAbrManager = class {
   }
 
 
+  /**
+   * @override
+   * @export
+   */
+  setPlaybackRate(rate) {
+    this.playbackRate_ = Number(rate);
```

Comment:
"rate" should already be a number on input.

Reply:
I changed it!